### DISCUSSION TITLE
Vercel deployment route inaccessible on iOS 14

### DISCUSSION THREAD

**Discussion by EfosaSO** [vercel/community#2634] [Bug Report]
Can't access any deployments on the web using my iPhone iOS 14.8.

Website just stays in constant loading state.

## Preview
https://github.com/vercel/community/assets/22990506/e0003417-8d8a-4b26-b234-d3badfa391fe

**Comment by swarnava** [comment]
Hi,

Could you confirm the version number you are using? Please note that Vercel is compatible only with iOS Safari v15.6 and later

**Reply by EfosaSO** [reply]
Ah, so compatibility is dropped for iOS versions below that?

**Reply by swarnava** [reply]
Correct. The site may still accessible, but we will likely not push any fix.
